tracing-subscriber/src/registry/stack.rs

```diff
@@ -8,7 +8,9 @@ struct ContextId {
     duplicate: bool,
 }
 
-/// `SpanStack` tracks what spans are currently executing on a per-thread basis.
+/// `SpanStack` tracks what spans are currently executing on a thread-local basis.
+///
+/// `SpanStack` is used as a performance optimization to reduce cross-thread traffic.
```

Comment:
I don't think this is true. I think having a separate current span for each thread is a _semantic_ choice, not strictly a perf one — each thread can be executing in a different thread.

Reply:
Added your note, thanks.